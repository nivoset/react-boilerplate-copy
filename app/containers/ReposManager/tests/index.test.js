/**
 *
 * Tests for ReposManager
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';

import ReposManager from '../index';
import configureStore from '../../../configureStore';

const renderReposManager = store =>
  render(
    <Provider store={store}>
      <ReposManager />
    </Provider>,
  );

describe('<ReposManager />', () => {
  let store;

  beforeEach(() => {
    store = configureStore({});
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = renderReposManager(store);
    expect(firstChild).toMatchSnapshot();
  });
});
