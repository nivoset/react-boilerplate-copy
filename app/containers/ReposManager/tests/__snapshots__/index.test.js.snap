// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReposManager /> should render and match the snapshot 1`] = `null`;
